examples/long_tutorials/handle_drift/plot_handle_drift.py

```diff
@@ -0,0 +1,342 @@
+#    be performed after motion correction.
+
+# %%
+# ----------------------------------------
+# Run motion correction with one function!
+# ----------------------------------------
+#
+# Correcting for drift is easy! You just need to run a single function.
+# We will now run motion correction on our recording using the three
+# presets described above - **rigid_fast**, **kilosort_like** and
+# **nonrigid_accurate**.
+#
+# We can run these presents with the ``preset`` argument of
+# :py:func:`~spikeinterface.preprocessing.correct_motion()`. Under the
+# hood, the presets define a set of parameters by set how to run the
+# 4 submodules that make up motion correction (described above).
+print(motion_options_preset["kilosort_like"])
+
+# %%
+# Now, lets run motion correction with our three presets. We will
+# set the ``job_kwargs`` to parallelize the job over a number of CPU cores—motion
+# correction is computationally intensive and will run faster with parallelization.
+
+presets_to_run = ("rigid_fast", "kilosort_like", "nonrigid_accurate")
+
+job_kwargs = dict(n_jobs=40, chunk_duration="1s", progress_bar=True)
+
+results = {preset: {} for preset in presets_to_run}
+for preset in presets_to_run:
+
+    corrected_recording, motion_info = si.correct_motion(
+        preprocessed_recording, preset=preset,  output_motion_info=True, **job_kwargs
+    )
+    results[preset]["motion_info"] = motion_info
+
+# %%
+#.. seealso::
+#   It is often very useful to save ``motion_info`` to
+#   file, so it can be loaded and visualised later. This can be done by setting
+#   the ``folder`` argument of
+#   :py:func:`~spikeinterface.preprocessing.correct_motion()` to a path to write
+#   the motion output to. The ``motion_info`` can be loaded back with
+#   ``si.load_motion_info``.
+
+# %%
+# -------------------
+# Plotting the results
+# -------------------
+#
+# Next, let's plot the results of our motion estimation using the ``plot_motion_info()``
+# function. The plot contains 4 panels, on the x-axis of all plots we have
+# the (binned time). The plots display:
+#   * **top left:** The estimated peak depth for every detected peak.
+#   * **top right:** The estimated peak depths after motion correction.
+#   * **bottom left:** The average motion vector across depths and all motion across spatial depths (for non-rigid estimation).
+#   * **bottom right:** if motion correction is non rigid, the motion vector across depths is plotted as a map, with the color code representing the motion in micrometers.
+
+for preset in presets_to_run:
+
+    fig = plt.figure(figsize=(7, 7))
+
+    si.plot_motion_info(
+        results[preset]["motion_info"],
+        recording=corrected_recording,  # the recording is only used to get the real times
+        figure=fig,
+        depth_lim=(400, 600),
+        color_amplitude=True,
+        amplitude_cmap="inferno",
+        scatter_decimate=10,            # Only plot every 10th peak
```

Comment:
Nice. This is another "term" that I think biologists/basic neuroscientists likely won't know so I like this comment a lot!